<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpPrinterPortWidget</class>
 <widget class="QWidget" name="TcpPrinterPortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>38</horstretch>
    <verstretch>44</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="ImportGroupBox">
     <property name="title">
      <string>Import</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="browseLabel">
        <property name="text">
         <string>Select Browse to import the settings for a given port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="browseToolButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="protocolGroupBox">
     <property name="title">
      <string>Protocol</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QRadioButton" name="tcpRadioButton">
        <property name="text">
         <string>TCP/IP (Raw)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="lprRadioButton">
        <property name="text">
         <string>LPR</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="rawGroupBox">
     <property name="title">
      <string>Raw Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Port Number:</string>
        </property>
        <property name="buddy">
         <cstring>portSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="specialValueText">
         <string>9100</string>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="lprGroupBox">
     <property name="title">
      <string>LPR Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="queueLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Queue Name:</string>
        </property>
        <property name="buddy">
         <cstring>queueLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="queueLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>26</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="lprCheckBox">
        <property name="text">
         <string>LPR Byte Counting Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="snmpCheckBox">
     <property name="text">
      <string>SNMP Status Enabled</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="communityLabel">
        <property name="text">
         <string>Community Name:</string>
        </property>
        <property name="buddy">
         <cstring>communityLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="communityLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="snmpLabel">
        <property name="text">
         <string>SNMP Device Index:</string>
        </property>
        <property name="buddy">
         <cstring>snmpSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="snmpSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>browseToolButton</tabstop>
  <tabstop>tcpRadioButton</tabstop>
  <tabstop>lprRadioButton</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>queueLineEdit</tabstop>
  <tabstop>lprCheckBox</tabstop>
  <tabstop>snmpCheckBox</tabstop>
  <tabstop>communityLineEdit</tabstop>
  <tabstop>snmpSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>tcpRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lprGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lprRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>rawGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>snmpCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
